src/schemathesis/schemas.py

```diff
@@ -82,6 +84,7 @@ def get_all_tests(
         """Generate all endpoints and Hypothesis tests for them."""
         test: Union[Callable, InvalidSchema]
         for endpoint in self.get_all_endpoints():
+            endpoint.wsgi_client = self.wsgi_client
```

Comment:
I think it should be where the `endpoint` is actually created, in `make_endpoint`